book-3-workforce/chapters/SQL_EXERCISES.md

```diff
@@ -0,0 +1,18 @@
+# SQL Exercises
+
+As you know SQL is a _very_ important skill for a precessional software developer to master. Frankly, you just can't get enough practice...so here's some more!
+
+Please go through the [SQL exercises on w3schools](https://www.w3schools.com/SQl/exercise.asp).
+
+Below the links to each exercise question (on the left side of the page) you'll find a link to a tutorial related the questions. If you are unclear about something click that link and view the tutorial.
```

Comment:
Below the links to each exercise question (on the left side of the page) you'll find a link to a tutorial related to the questions. If you are unclear about something, click that link and view the tutorial.